docs/Benodigde-tabellen.md

```diff
@@ -0,0 +1,20 @@
+Noot: De landelijke tabellen zijn gepubliceerd op https://publicaties.rvig.nl/Landelijke_tabellen/Landelijke_tabellen_32_t_m_61_excl_tabel_35
+
+Voor de BRP-Personen-Bevragen moeten minimaal de volgende tabellen beschikbaar worden gemaakt:
+
+Aanduiding naamgebruik (Waardelijst uit het LO GBA Element 61.10 Aanduiding naamgebruik)
+Aanduiding bij huisnummer (Waardelijst uit het LO GBA Element 11.50 Aanduiding bij huisnummer)
+Aanduiding bijzonder Nederlanderschap (Waardelijst uit het LO GBA Element 65.10 Aanduiding bijzonder Nederlanderschap)
+Aanduiding bijzonder Nederlanderschap
+AdellijkeTitelPredicaat ( Landelijke tabel 38)
+FunctieAdres (Waardelijst uit het LO GBA Element 10.10 Functie Adres)
+Gemeenten (Landelijke tabel 33)
+Geslachtsaanduiding (Waardelijst uit het LO GBA 3.14 Element 04.10 Geslachtsaanduiding)
+Indicatie gezag minderjarige (Landelijke tabel 61)
```

Comment:
waarom is deze eigenlijk ook in tabelwaarden.feature opgenomen?

Reply:
de naam van de tabel is "Gezagsverhouding"

Reply:
> waarom is deze eigenlijk ook in tabelwaarden.feature opgenomen?

Omdat deze in indicatieGezagMinderjarige wordt gebruikt.

Reply:
> > waarom is deze eigenlijk ook in tabelwaarden.feature opgenomen?
> 
> Omdat deze in indicatieGezagMinderjarige wordt gebruikt.

ik heb hiervoor issue #1065 gemaakt